docs/includes/notes/protocol_oidc.mdx

```diff
@@ -0,0 +1,11 @@
+<details>
```

Comment:
@patriciamcphee, Is there a bast practice around Collapsible Text Blocks? I thought it was cleaner for these particular elements to be collapsible, but if that is a big no, then I can revert these back to the Info Admonition.

Reply:
For this purpose, they're perfect.

Also, on your include files, make sure it starts with an underscore.